app/src/features/apiClient/screens/PostmanImporterView/components/PostmanImporter/utils.ts

```diff
@@ -81,6 +83,24 @@ const processScripts = (item: any) => {
   return scripts;
 };
 
+const processAuthorizationOptions = (item = {}) => {
+  try {
+    const auth = {};
+    const authType = POSTMAN_AUTH_TYPES_MAPPING[item.type] ?? AUTHORIZATION_TYPES.NO_AUTH;
+    auth.currentAuthType = authType;
+    auth[authType] = {};
+
+    const authOptionsArray = item[item?.type] || [];
+    authOptionsArray.forEach((option) => {
+      auth[authType][option.key] = option.value;
+    });
+
+    return auth;
+  } catch (error) {
+    return {};
+  }
+};
+
```

Comment:
Please try to not mix features as possible in single PR. This should be in a separate PR that builds on top of this.